backend/spec/features/admin/orders/cancelling_and_resuming_spec.rb

```diff
@@ -39,7 +39,7 @@
       click_button 'Resume'
       within(".additional-info") do
         within(".state") do
-          expect(page).to have_content("resumed")
+          expect(page).to have_content("complete")
```

Comment:
better to use 'complete'.freeze (not mandatory) freeze will retain this string object. and it is preferred to use double quotation when interpolation is needed.

Reference: https://github.com/bbatsov/ruby-style-guide#consistent-string-literals